bugsnag-android-core/src/test/java/com/bugsnag/android/EventFacadeTest.java

```diff
@@ -183,4 +184,11 @@ public void setUserValid() {
         event.setUser(null, null, null);
         assertEquals(new User(null, null, null), event.getUser());
     }
+
+    @Test
+    public void unhandledValid() {
+        assertFalse(event.isUnhandled());
+        event.setUnhandled(true);
+        assertTrue(event.isUnhandled());
```

Comment:
For completeness, wouldn't we set it back to false to make sure you can toggle both ways?